docs/usage/known-limitations.md

```diff
@@ -0,0 +1,33 @@
+# Known limitations
+
+Learn about the limitations of Renovate bot.
+
+## Introduction
+
+Renovate, like any computer program has limitations.
+It's good to remember that Renovate takes time to process your repository.
+Processing time scales with repository size and complexity.
+This means that a simple repository with just one `package.json` file with 3 dependencies in it will be done faster than a big monorepo with 20 `package.json` files with lots of dependencies.
+
+Do not panic if Renovate doesn't react immediately when you click on a checkbox to rebase your branch, or create a branch.
+
+It's usually a good idea to wait 2 hours before starting troubleshooting.
+Read our [guide on troubleshooting](https://docs.renovatebot.com/troubleshooting/) if you're having problems.
+
+## Time based limitations
+
+Renovate can only run as often/long as the admin has set it to.
+For the GitHub hosted app, this means that Renovate will do a run each hour by default.
+
+Renovate will also only process the repository in the schedule that is set for it.
+
+## Automerge limitations
+
+Renovate can only merge one branch per run.
+Renovate can only automerge when the update branch is up-to-date with the target branch.
+Renovate connot automerge if you or others keep committing to the target branch.
+
+What all this means is that generally, Renovate will only automerge one update at a time.
+
+This limitation is because we want Renovate to be up-to-date with the target branch before automerging.
+This prevents problems when a change on the target branch renders the update incompatible.
```

Comment:
```suggestion
The limitation to only merge one branch per run is because Renovate's dependency and branch state is based on what was present in the base branch at the start of the run.
If a branch is merged into the base branch during Renovate's run - including by other users - it means that remaining Renovate branches may have git conflicts. It also means that Renovate's knowledge about dependencies in the base branch is now invalid and other branches may need changing as a result of the merge.

The limitation to only automerge branches which are up-to-date is a decision due to this example:
- Two dependencies are in use: `a@1.0.0` and `b@1.0.0`
- PRs exist for `a@2.0.0` and `b@2.0.0` and both pass tests
- The PR for `a@2.0.0` is automerged
- The PR for `b@2.0.0` remains open, does not have conflicts, and has all tests passing
- However, `a@2.0.0` and `b@2.0.0` are incompatible so merging the PR without rebasing and retesting it first would result in a broken base branch
```